desc/objectives/objective_funs.py

```diff
@@ -481,6 +481,7 @@ def __init__(
         assert np.all(np.asarray(weight) > 0)
         assert normalize in {True, False}
         assert normalize_target in {True, False}
+        assert len(target) == 2 if isinstance(target, tuple) else True
```

Comment:
I could see a list being passed in here too, do we want to check if it is a list and convert to tuple if so before doing this assertion?

Reply:
Lists are already used for targets in the original sense (previous to this PR), where they get cast to jnp arrays. So giving the target as a tuple is the only allowable way to indicate that it is meant to be lower/upper bounds on the target.

Reply:
ok that makes sense,